<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>1081</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x0spinBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>66</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x1spinBox">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>390</y>
      <width>66</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="t0spinBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>66</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="t1spinBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>66</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>16</width>
      <height>281</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="calcButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>410</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cntSpinBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>410</y>
      <width>47</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1227</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
